mlflow/tracking/utils.py

```diff
@@ -198,11 +182,60 @@ def get_db_profile_from_uri(uri):
     return None
 
 
-def _get_databricks_rest_store(store_uri):
+def _get_databricks_rest_store(store_uri, artifact_uri):  # pylint: disable=W0613
     profile = get_db_profile_from_uri(store_uri)
     return RestStore(lambda: get_databricks_host_creds(profile))
 
 
+class TrackingStoreRegistry:
```

Comment:
Mind adding a brief class comment on what this is responsible for?

Reply:
Done, please take a look to see if they are clear enough.